addons/fleet/models/fleet_vehicle_cost.py

```diff
@@ -140,7 +140,7 @@ def default_get(self, default_fields):
         })
         return res
 
-    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', default=1, required=True, help='Vehicle concerned by this log')
+    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', required=True, help='Vehicle concerned by this log')
```

Comment:
There is the same `default=1` on line 20. Maybe we should fix it in the same commit?

Reply:
Indeed, you are right :-) 
Thanks!